app/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">FreeOTP</string>
+    <string name="invalid_token">The token specified was invalid!</string>
+    <string name="delete">Удалить</string>
+    <string name="edit">Изменить</string>
+    <string name="restore_defaults">Восстановить значения по умолчанию</string>
+    <string name="no_keys">Не установлены OTP-ключи.</string>
+    <string name="add_token">Добавить токен</string>
+    <string name="scan_qr_code">Сканировать QR код</string>
+    <string name="interval">Интервал</string>
+    <string name="counter">Счетчик</string>
+    <string name="secret">Секрет</string>
+    <string name="type">Тип</string>
+    <string name="algorithm">Алгоритм</string>
+    <string name="digits">Цыфр</string>
```

Comment:
I suppose it's should be ```Цифр``` .

http://gramota.ru/class/coach/tbgramota/45_75

Reply:
D'oh! Thank you.